tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -219,31 +242,19 @@ export class Ide {
     }
 
     async closeAllNotifications() {
-        const notificationLocator: By = By.css('.theia-Notification');
-
+        const statusBarNotificationsLocator: By = By.xpath(`//div[@id="theia-statusBar"]/div[@class="area right"]//div[contains(@title, 'Notification')]`);
+        const notificationCenterActionCloseAllButtonLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center open')]//ul[@class="theia-notification-actions"]/li[@class="clear-all"]`);
+        const notificationCenterClosedLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center closed')]`);
         Logger.debug('Ide.closeAllNotifications');
 
-        if (! await this.driverHelper.isVisible(notificationLocator)) {
-            return;
-        }
-
-        const notifications: WebElement[] = await this.driverHelper.waitAllPresence(notificationLocator);
-        const notificationsCapacity: number = notifications.length;
+        Logger.debug(`Ide.closeAllNotifications driverHelper click Notification Center icon.`);
```

Comment:
Here and everywhere please do not use logging too much.
It is pretty enough ```Logger.debug('Ide.closeAllNotifications');```
If you want to log inner steps (if they don't consist of dedicated methods) use ```Logger.trace```

Reply:
Thank you, will change it to trace

Reply:
done.